### DISCUSSION TITLE
Android: do I need to call static method LogManager.initialize(String tenantToken, ILogConfiguration configuration)

### DISCUSSION THREAD

**Discussion by theta3** [microsoft/cpp_client_telemetry#1030] [General]
Looking at section 6 https://github.com/microsoft/cpp_client_telemetry/blob/main/docs/cpp-start-android.md. `ILogger` is obtained through calling `LogManager.initialize()`. But there is also `static LogManager.getLogger(tenantId, source)` and non static `getLogger()`. Looking at the test https://github.com/microsoft/cpp_client_telemetry/blob/main/lib/android_build/app/src/androidTest/java/com/microsoft/applications/events/maesdktest/LogManagerDDVUnitTest.java : `multipleLogManagerInstantiation()`, `initialize()` isn't called. What's the difference between these 3 methods?

Do I need to call `LogManager.initialize()` to init the SDK?

If I create multiple `ILogConfiguration` configs, which config will be used when calling  `static LogManager.getLogger(tenantId, source)` ?

**Comment by lalitb** [comment]
@saurabh-sagrawal, @larvacea  - Can you have a look at this ask please?
